{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/globals.py

```diff
@@ -12,6 +12,7 @@
 MAKE_ARTIFACT_MEM = '3G'
 MAKE_ARTIFACT_CPU = '1'
 MAKE_ARTIFACT_RUNTIME = '3:00:00'
+MAKE_ARTIFACT_SLEEP = 10
```

Comment:
:+1:

Reply:
What's this for ? I don't see it used

Reply:
Umm, good catch -- this is used in the code I initially checked in but stubbed out in the follow up. It will show up again with the next PR. I will pull it from this and add back with the code that uses it.